packages/edge-config/src/index.ts

```diff
@@ -81,6 +83,49 @@ const getFileSystemEdgeConfig = trace(
   },
 );
 
+/**
+ *
+ */
+const getPrivateEdgeConfig = trace(
+  async function getPrivateEdgeConfig(
+    connection: Connection,
+  ): Promise<DeepReadonly<EmbeddedEdgeConfig> | null> {
+    const privateEdgeConfig = Reflect.get(
```

Comment:
What's the advantage of `Reflect.get` here over `globalThis[privateEdgeConfigSymbol]`?

Reply:
@dferber90 no need to cast `globalThis`, just the returned type.